### PR TITLE
Shooter hardware

### PR DESCRIPTION
**Pull request by minhnguyenbhs**
Shooter hardware implementation- testing on robot

### CODE REVIEW

**Comment by minhnguyenbhs** [comment]
@aidnem hi!! do you know if shop tomorrow is confirmed from 1-4pm? I can't come during that time but if shop is open at any other time on Saturday I can come to test stuff on the robot :DDDD
if it is confirmed exactly 1-4pm could you maybe test out the shooter code if you have time? thanks!!

**Comment by aidnem** [comment]
@minhnguyenbhs yes shop is confirmed for 1-4PM as far as I know. Sorry you can't come. I'm still working on sorting out priorities for integration but if I have time I would be glad to test it out :). You could also try it on Monday if I don't get to it.

Would just opening up scope and testing to see if the banner sensor is working be helpful for you even if I can't test the full subsystem?

**Comment by minhnguyenbhs** [comment]
ok thank you!! yeah I'll get to it Monday if needed :DDD that would be helpful but probably isn't a priority, I can wait to test it on Monday! thanks!

**Comment by jkleiber** [comment]
@minhnguyenbhs this branch has a ton of merge conflicts (and will have more once the work from yesterday gets merged). Can we clean this up before Wednesday's meeting?

**Comment by PChild** [comment]
This needs to be updated: https://github.com/team401/high-key-2024/blob/d2b9aba8c803bc4df11e451da45f9af136f0d886/src/main/java/frc/robot/constants/ScoringConstants.java#L48

As mentioned in discussion #26  the kicker is CAN ID 5 now.

**Comment by PChild** [comment]
@minhnguyenbhs There is only one arm motor. The encoder is a cancoder that is 1:1 with the arm. I believe @aidnem is working on code to use [FusedCancoder](https://v6.docs.ctr-electronics.com/en/latest/docs/api-reference/device-specific/talonfx/remote-sensors.html#fusedcancoder) for it.

File: src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java [hunk 1]
```diff
@@ -429,6 +431,30 @@ public void periodic() {
...
+                SmartDashboard.getNumber("kV_Aimer", ScoringConstants.aimerkV),
+                SmartDashboard.getNumber("kA_Aimer", ScoringConstants.aimerkA),
+                SmartDashboard.getNumber("kG_Aimer", ScoringConstants.aimerkG),
+                0);
+        setFF(
+                SmartDashboard.getNumber("kS_Shooter", ScoringConstants.shooterkS),
+                SmartDashboard.getNumber("kV_Shooter", ScoringConstants.shooterkV),
+                SmartDashboard.getNumber("kA_Shooter", ScoringConstants.shooterkA),
+                0,
+                0);*/
```
Comment (jkleiber) [inline src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java:456]: Let's delete these for now

**Review by jkleiber — COMMENTED** [review]
just a few small nitpicks before we merge this

**Review by jkleiber — APPROVED** [review]

**Pull request merged**